src/main/types/SSI.types.ts

```diff
@@ -3,6 +3,7 @@ import { JWK } from 'jose';
 
 export interface ResolveOpts {
   resolver?: Resolvable;
+  resolvers?: Map<string, Resolvable>;
```

Comment:
TBH I am not sure about this approach. Probably we should place a todo here linking to the ticket recently created

Reply:
added todo for the other ticket